src/backend/data/post.js

```diff
@@ -66,6 +68,17 @@ class Post {
       article.title = 'Untitled';
     }
 
+    // If we're missing dates, assign current date
+    const today = new Date();
+    if (!article.pubdate) {
+      logger.debug('article missing pubdate, substituting current date');
+      article.pubdate = today;
+    }
+    if (!article.date) {
+      logger.debug('article missing date, substituting current date');
+      article.date = today;
+    }
+
     let sanitizedHTML;
     let plainText;
```

Comment:
I don't see us using these vars or am I wrong?

Reply:
Lines 87-89 below.  It's not shown with the default level of context, but it looks like this:

```js
let sanitizedHTML;
let plainText;

try {
  // The article.description is frequently the full HTML article content.
  // Sanitize it of any scripts or other dangerous attributes/elements
  sanitizedHTML = sanitizeHTML(article.description);
  // Also generate plain text from the sanitized HTML
  plainText = textParser(sanitizedHTML);
} catch (err) {
  logger.error({ err }, 'Unable to sanitize and parse HTML for feed');
  throw err;
}
```